pkg/ephemeral/io/tuple_streamer_test.go

```diff
@@ -0,0 +1,146 @@
+//
+// Copyright (c) 2022 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/carbynestack/ephemeral.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+package io
+
+import (
+  "sync"
+  "time"
+  "errors"
+  "math/big"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/carbynestack/ephemeral/pkg/castor"
+	. "github.com/carbynestack/ephemeral/pkg/types"
+	"go.uber.org/zap"
+  "github.com/google/uuid"
+)
+
+const tupleStock = 42
+
+var _ = Describe("Tuple Streamer", func() {
+  var (
+    pipeWriter *FakeConsumingPipeWriter
+    ts         CastorTupleStreamer
+    cc         *FakeCastorClient
+  )
+  BeforeEach(func() {
+    pipeWriter = &FakeConsumingPipeWriter{
+      isClosed: false,
+    }
+    cc = &FakeCastorClient{}
+    ts = CastorTupleStreamer{
+  		logger:        zap.NewNop().Sugar(),
+  		pipeWriter:    pipeWriter,
+      tupleType:     castor.BitGfp,
```

Comment:
Formatting/whitespaces seems off?
Logger and PipeWriter have a Tab character infront of them, tupleType only spaces.

Reply:
should be fixed now